app/models/UserAnswersSummary.scala

```diff
@@ -34,11 +36,12 @@ case class UserAnswersSummary(eoriNumber: String, userAnswers: Seq[UserAnswers],
             "_links" -> Json.obj(
               "self" -> Json.obj("href" -> controllers.routes.CacheController.get(userAnswer.lrn).url)
             ),
-            "createdAt"     -> userAnswer.createdAt,
-            "lastUpdated"   -> userAnswer.lastUpdated,
-            "expiresInDays" -> expiresInDays(userAnswer.createdAt),
-            "_id"           -> userAnswer.id,
-            "isSubmitted"   -> userAnswer.status
+            "createdAt"      -> userAnswer.createdAt,
+            "lastUpdated"    -> userAnswer.lastUpdated,
+            "expiresInDays"  -> expiresInDays(userAnswer.createdAt),
+            "_id"            -> userAnswer.id,
+            "isSubmitted"    -> userAnswer.status,
+            "isTransitional" -> userAnswer.isTransitional.getOrElse(DefaultIsTransitionalFlag)
```

Comment:
This would then become `userAnswer.isTransitional` and `DefaultIsTransitionalFlag` could be removed